frontend/packages/pipelines-plugin/src/utils/pipeline-filter-reducer.ts

```diff
@@ -2,22 +2,39 @@ import * as _ from 'lodash';
 
 export const pipelineRunStatus = (pipelineRun): string => {
   const conditions = _.get(pipelineRun, ['status', 'conditions'], []);
+  if (conditions.length === 0) return null;
   const isCancelled = conditions.find((c) =>
-    ['PipelineRunCancelled', 'TaskRunCancelled'].some((cancel) => cancel === c.reason),
+    ['PipelineRunCancelled', 'TaskRunCancelled', 'Cancelled'].some((cancel) => cancel === c.reason),
   );
   if (isCancelled) {
     return 'Cancelled';
   }
```

Comment:
Hmmm. I know its saver to keep the existing code. But what do you think if we move this (similar to the go code) also in the switch below?